src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesPairs = sourceString.split(';');
```

Comment:
[CODE PRACTICE] - To improve the readability and maintainability of the code, consider chaining the operations instead of creating intermediate variables like `stylesPairs` and `stylesArray`. You can chain the `split`, `map`, and `trim` operations directly, which will also address the checklist item regarding unnecessary copies of arrays.